osbs/api.py

```diff
@@ -681,7 +681,7 @@ def create_orchestrator_build(self, **kwargs):
 
     def _decode_build_logs_generator(self, logs):
         for line in logs:
-            line = line.decode("utf-8")
+            line = line.decode("utf-8").strip()
```

Comment:
In some cases, I'm getting newline characters along with the line, so the strip was necessary to get the output fully sanitized.